concepticondata/conceptlists/Morse-1996-375.tsv

```diff
@@ -0,0 +1,376 @@
+ID	NUMBER	SPANISH	ENGLISH	SWADESH_ROWE	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Morse-1996-375-1	1	lengua	tongue	*	1205	TONGUE
+Morse-1996-375-2	2	boca	mouth	*	674	MOUTH
+Morse-1996-375-3	3	labio	lip	*	478	LIP
+Morse-1996-375-4	4	diente	tooth	*	1380	TOOTH
+Morse-1996-375-5	5	nariz	nose	*	1221	NOSE
+Morse-1996-375-6	6	ojo	eye	*	1248	EYE
+Morse-1996-375-7	7	oreja	ear	*	1247	EAR
+Morse-1996-375-8	8	cabeza	head	*	1256	HEAD
+Morse-1996-375-9	9	frente	forehead	*	123	FOREHEAD
+Morse-1996-375-10	10	cabello	hair	*	2648	HAIR (HEAD)
+Morse-1996-375-11	11	mentón	chin		1510	CHIN
+Morse-1996-375-12	12	barba	beard		678	BEARD
+Morse-1996-375-13	13	cuello	neck	*	1333	NECK
+Morse-1996-375-14	14	pecho	chest	*	1592	CHEST
+Morse-1996-375-15	15	teta, seno, pechos	woman’s breast		1402	BREAST
+Morse-1996-375-16	16	vientre, abdomen	abdomen			
```

Comment:
Could it be mapped to 1251 BELLY?

Reply:
I'd leave this one unmapped because the Spanish _vientre_ seems to mean "womb" and there is another word mapped to BELLY in the list.

Reply:
Yes, and there also is another word mapped to STOMACH, so I'm not sure at all which concept they wanted to go for here.

Reply:
This has an archaic touch, but if there's no BELLY, you can map to belly.